src/styles/index.scss

```diff
@@ -1,3 +1,115 @@
+$main-accent: #060b35;
+$secondary: #616070;
+$blue-accent: #00acdc;
+
 body {
   margin: 0;
 }
+
+p {
+  margin: 0;
+}
+
+.product-card {
+  box-sizing: border-box;
+  font-family: Roboto, Arial, sans-serif;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  width: 200px;
+  padding: 206px 16px 16px;
+  background-image: url(../images/imac.jpeg);
+  background-size: 160px 134px;
+  background-repeat: no-repeat;
+  background-position: center 32px;
```

Comment:
Better to provide image with img tag cause this easier to control changes and if you img will change that`s will be better too, so please remove it and make the same but using img tag